internal/command/jobs/printers.go

```diff
@@ -0,0 +1,188 @@
+		}
+
+		t.AddRow(jobRow{
+			Name:        job.Name,
+			Environment: environment,
+			StartedAt:   startedAt,
+			Duration:    duration,
+			Status:      job.Status.Phase,
+		})
+	}
+	return t.Flush()
+}
+
+func printJobDetails(cfg *config.Job, out io.Writer, job *models.JobsJob) error {
+	tw := tabwriter.NewWriter(out, 0, 0, 3, ' ', 0)
+
+	startedAt, duration := getStartedAndDuration(job)
+
+	fmt.Fprintf(tw, "Job Name:\t%s\n", job.Spec.NamePrefix)
+	fmt.Fprintf(tw, "Generated Job Name:\t%s\n", job.Name)
+	fmt.Fprintf(tw, "Status:\t%s\n", job.Status.Phase)
+	fmt.Fprintln(tw)
+
+	fmt.Fprintf(tw, "Environment:\t%s\n", getJobEnvironment(job))
+	fmt.Fprintf(tw, "Started At:\t%s\n", startedAt)
+	fmt.Fprintf(tw, "Duration:\t%s\n", duration)
+
+	if err := printArtifacts(cfg, tw, job); err != nil {
+		return err
+	}
+
+	if err := tw.Flush(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func getStartedAndDuration(job *models.JobsJob) (startedAtStr string, durationStr string) {
+	var startedAt *time.Time
+
+	if len(job.Status.Attempts) == 0 {
+		return "", ""
+	}
+
+	attempt := job.Status.Attempts[0]
+
+	if attempt.Phase == "queued" {
+		return "", ""
+	}
+
+	startedAtRaw := attempt.CreatedAt
+	if len(startedAtRaw) != 0 {
+		t, err := time.Parse(time.RFC3339, startedAtRaw)
+		if err != nil {
+			return "", ""
+		}
+
+		startedAt = &t
+		startedAtStr = timeago.NoMax(timeago.English).Format(t)
+	}
+
+	finishedAtRaw := attempt.FinishedAt
+	if startedAt != nil && len(finishedAtRaw) != 0 {
+		finishedAt, err := time.Parse(time.RFC3339, finishedAtRaw)
+		if err != nil {
+			return "", ""
+		}
+
+		durationTime := finishedAt.Sub(*startedAt)
+		durationStr = durationTime.String()
+	}
+
+	return startedAtStr, durationStr
+}
+
+func getJobEnvironment(job *models.JobsJob) string {
+	routingContext := job.Spec.RoutingContext
+
+	if routingContext == nil {
+		return "BASELINE"
+	}
+
+	if len(routingContext.Sandbox) > 0 {
+		return fmt.Sprintf("%s (SANDBOX)", routingContext.Sandbox)
+	}
+
+	return fmt.Sprintf("%s (ROUTEGROUP)", routingContext.Routegroup)
+}
+
+func getArtifacts(cfg *config.Job, job *models.JobsJob) ([]*models.JobArtifact, error) {
+	params := artifacts.NewListJobAttemptArtifactsParams().
+		WithOrgName(cfg.Org).
+		WithJobAttempt(job.Status.Attempts[0].ID).
+		WithJobName(job.Name)
+
+	resp, err := cfg.Client.Artifacts.ListJobAttemptArtifacts(params, nil)
+	if err != nil {
+		return []*models.JobArtifact{}, nil
+	}
+
+	return resp.Payload, nil
+}
+
+type jobArtifactRow struct {
+	Name string `sdtab:"NAME"`
+	Url  string `sdtab:"URL"`
+}
+
+func printArtifacts(cfg *config.Job, out io.Writer, job *models.JobsJob) error {
+	artifactsList, err := getArtifacts(cfg, job)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(out, "\nArtifacts\n")
+
+	if len(artifactsList) == 0 {
+		fmt.Fprintln(out, "No artifacts")
+		return nil
+	}
+
+	t := sdtab.New[jobArtifactRow](out)
+	t.AddHeader()
+	for _, artifact := range artifactsList {
```

Comment:
I think we need not print each individual URL here at all - we can just print the dashboard URL once under `job get` and leave just the names of the artifact items here. We will use the `signadot artifact` command to enable downloading these entities anyway, so, the individual URLs don't help.